BodoSQL/bodosql/context_ext.py

```diff
@@ -174,7 +174,7 @@ def box_bodosql_context(typ, val, c):
         )
 
     mod_name = c.context.insert_const_string(c.builder.module, "bodosql")
-    bodosql_class_obj = c.pyapi.import_module_noblock(mod_name)
+    bodosql_class_obj = c.pyapi.import_module(mod_name)
```

Comment:
Could we not abstract this somewhere in Bodo so that future changes are isolated to one line rather than the 50 changes we have in this PR?

Reply:
This is a very rare change and I don't expect it to happen again. Their API name was just stuck on very old Python.